### DISCUSSION TITLE
Where to look for documentation?

### DISCUSSION THREAD

**Discussion by pmbanugo** [quickjs-ng/quickjs#495] [Q&A] [answered]
Hi, I'm curious to try this out. It looks more active compared to [the original from bellard](https://github.com/bellard/quickjs), which points to this documentation.: https://bellard.org/quickjs/

However, I want to try this one since it seems to have moved much further in terms of commits and ES compatibility. Where can I red the docs to figure out how to use this as an embeddable JS engine? Or is [tixki](https://github.com/saghul/txiki.js) the recommended path? I think tixki isn't embeddable but I couldn't figure out what it should be used for.

**Comment by saghul** [comment, marked as answer]
Hey there! We haven't yet created new documentation.

Most of that documentation still stands though. While we added a bunch of JS APIs and things, the core QuickJS API is the same. If you want to embed it into your project you can follow that.

As the author of txiki.js: it's not currently designed to be embedded but rather to be a standalone, minimalistic runtime.

**Reply by pmbanugo** [reply]
Thank you.

**Comment by pmbanugo** [comment]
is there a way to compile it to output dynamic loadable shared library? .so for linux and mac?

I'm looking for a means to compile into a dynamic loadable shared library (.SO in Unix, DLL in Windows), with possibility of calling into the functions in the shared library.

**Reply by bnoordhuis** [reply]
Yes, `cmake -DBUILD_SHARED_LIBS=ON`
